### PR TITLE
Pr/cropper text label support

### PR DESCRIPTION
**Pull request by bhagyae5308**
The changes are related to #381 
- Added a helper text on top of crop overlay which moves along with it
- Exposed xml attrs and setter to update the copy text styles

Adding sample working video after the change

https://user-images.githubusercontent.com/8956142/172991936-163c75f4-2fa5-4fe8-b6fc-c25002a4eec9.mp4

### CODE REVIEW

File: cropper/src/main/java/com/canhub/cropper/CropOverlayView.kt [hunk 1]
```diff
@@ -34,6 +34,18 @@ class CropOverlayView
...
+         */
+        private fun getTextPaint(options: CropImageOptions): Paint {
+            return Paint().apply {
+                strokeWidth = 1f
+                textSize = options.cropperLabelTextSize
+                style = Paint.Style.FILL
+                textAlign = Paint.Align.CENTER
+                this.color = options.cropperLabelTextColor
+            }
+        }
```
Comment (Canato) [inline cropper/src/main/java/com/canhub/cropper/CropOverlayView.kt:48]: Please change to:
`/** Creates the paint object for drawing text label over crop overlay */`

`private fun getTextPaint(options: CropImageOptions): Paint = `, removing `return`

File: cropper/src/main/java/com/canhub/cropper/CropOverlayView.kt [hunk 2]
```diff
@@ -92,6 +104,8 @@ class CropOverlayView
     /** The Paint used to darken the surrounding areas outside the crop area.  */
     private var mBackgroundPaint: Paint? = null
 
+    private var mTextLabelPaint: Paint? = null
```
Comment (Canato) [inline cropper/src/main/java/com/canhub/cropper/CropOverlayView.kt:107]: this `m` is the old java way. And we didn't remove every where, please use `textLabelPaint`

**Review by Canato — CHANGES_REQUESTED** [review]
Amazing work!

Only comments are related to stuff you could not know before anyway, some old Java way of doing we are trying to improve.

Please update the `CHANGELOG`

File: sample/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -20,7 +20,7 @@
         <activity android:name="com.canhub.cropper.sample.SampleResultScreen" />
         <activity android:name="com.canhub.cropper.CropImageActivity" />
         <activity
-            android:name="com.canhub.cropper.sample.extend_activity.app.SExtendActivity"
+            android:name="com.canhub.cropper.sample.SampleCustomActivity"
```
Comment (bhagyae5308) [inline sample/src/main/AndroidManifest.xml:23]: @Canato It was an existing crash, so fixed it along with other changes

**Review by Canato — APPROVED** [review]
Perfect!

**Pull request merged**